packaging/targets/trusty/debian/rules

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/make -f
+# Uncomment this to turn on verbose mode.
+export DH_VERBOSE=1
+
+REVISION := $(shell head -1 debian/changelog | sed 's/.*(//;s/).*//;s/.*-//')
+
+build: build-stamp
+build-stamp:
+	dh_testdir
+	touch build-stamp
+
+clean: 
+	dh_testdir
+	dh_testroot
+	rm -f build-stamp
+	dh_clean 
+
+install: build
```

Comment:
Could we make the install target just do a `npm install openhim-core -g`. This could simplify things greatly as we don't have to  deal with the source at all?